galaxy.yml

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 ---
-version: 1.2.7
```

Comment:
Since this PR is adding a new module now, we'll want to increment the minor version so this will be `1.3.0`.